Header header
int32 user_id
string username
user_tracker/ArmPos left_arm
user_tracker/ArmPos right_arm
user_tracker/LegPos left_leg
user_tracker/LegPos right_leg
user_tracker/JointPos head
user_tracker/JointPos neck
user_tracker/JointPos torso
JointAng body
